GOF/app/views/houses/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<head><title>Editing Dope House...</title></head>
+
+<% @house.errors.full_messages.each do |message| %>
+  <p><%= message %></p>
+<% end %>
+
+<h1> Edit House </h1>
+<%= form_for @house do |t| %>
+<%= t.label :House_name %>
+<%= t.text_field :house_name %>
+
+<%= t.label :Kingdom %>
+<%= t.text_field :kingdom %>
+
+<%= t.label :Number_Of_Slaves %>
+<%= t.text_field :numOfSlaves %>
+
+<%= t.label :Image  %>
+<%= t.text_field :img_url %>
+
+<%= t.submit %>
+<% end %>
```

Comment:
looks good, but you can also use a partial for this form and the new form, so you dont have to type out the same form over and over again